internal/cli/atlas/privateendpoints/interfaces/describe.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package interfaces
+
+import (
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/output"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+type DescribeOpts struct {
+	cli.GlobalOpts
+	id                string
+	privateEndpointID string
+	store             store.InterfaceEndpointDescriber
+}
+
+func (opts *DescribeOpts) init() error {
+	var err error
+	opts.store, err = store.New(config.Default())
+	return err
+}
+
+var describeTemplate = `ID	STATUS	ERROR
+{{.ID}}	{{.ConnectionStatus}}	{{.ErrorMessage}}
+`
+
+func (opts *DescribeOpts) Run() error {
+	r, err := opts.store.InterfaceEndpoint(opts.ConfigProjectID(), opts.privateEndpointID, opts.id)
+
+	if err != nil {
+		return err
+	}
+
+	return output.Print(config.Default(), describeTemplate, r)
+}
+
+// mongocli atlas privateEndpoint(s) interface(s) describe <interfaceEndpointId> [--privateEndpointId privateEndpointID][--projectId projectId]
+func DescribeBuilder() *cobra.Command {
+	opts := new(DescribeOpts)
+	cmd := &cobra.Command{
+		Use:     "describe",
+		Aliases: []string{"get"},
+		Args:    cobra.ExactArgs(1),
+		Short:   describeInterfaceEndpoint,
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			opts.id = args[0]
+			return opts.PreRunE(opts.init)
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return opts.Run()
+		},
+	}
+	cmd.Flags().StringVar(&opts.privateEndpointID, flag.PrivateEndpointID, "", usage.PrivateEndpointID)
+
+	cmd.Flags().StringVar(&opts.ProjectID, flag.ProjectID, "", usage.ProjectID)
+
```

Comment:
I think `PrivateEndpointID` is required [doc](https://docs.atlas.mongodb.com/reference/api/private-endpoint-get-one-interface-endpoint/)